R/createReport.R

```diff
@@ -768,6 +769,50 @@ createReport = function(txt_folder = NULL, mztab_file = NULL, yaml_obj = list(),
   #save(file = rprt_fns$R_plots_file, list = "GPL")
   #cat(" done\n")
   
+  ##save required data of each metric in JSON-format
+  cat("Creating metric json file ...", '\n')
+  ##load the cv Term
+  cv <- read_cvTerm(txt_files$cv)
+  ##copy cv Term table
```

Comment:
why copy the table again? `cv` is not used anymore, right? So why not
`metric_info  = read_cvTerm(cv_obo_file )`

and do the colnames and conversion to data.frame already inside the `read_cvTerm` function.